agent/util-scripts/pbench-tool-meister-start

```diff
@@ -353,6 +356,17 @@ def main(argv):
         ), f"bad channel: {resp!r}"
         assert resp["data"] == 1, f"bad data: {resp!r}"
 
+    # 2.5. Add tool metadata json to redis
+    try:
+        inst_dir = PbenchAgentConfig(os.environ["_PBENCH_AGENT_CONFIG"]).pbench_install_dir
+        tm_start_path = Path(inst_dir).resolve(strict=True)
+    except FileNotFoundError:
+        logger.error("Unable to determine proper installation directory, {inst_dir} not found")
+    except Exception as exc:
+        loger.exception(f"Failed to resolve expected installation: {exc}")
```

Comment:
`s/loger/logger/`

And consider using a message like the following without emitting the exception stack trace, like:

```
logger.error("Unexpected error encountered resolving installation directory: '%s'", exc)
```